pkg/helper/client.go

```diff
@@ -96,7 +85,7 @@ func UpdateCurrentState(client client.Client, nodeNetworkState *nmstatev1alpha1.
 		return fmt.Errorf("error running nmstatectl show: %v", err)
 	}
 	observedState := nmstatev1alpha1.State{Raw: []byte(observedStateRaw)}
-
+	interfacesFilterGlob := glob.MustCompile(configurations.GetCurrentConfig().InterfaceFilter)
```

Comment:
We can compile it once in before we call the config setter.

Reply:
U want me to add it to configurations? I think it will be a little strange but it's ok

Reply:
Meh, no need to.

Reply:
No it's ok. This function is called A lot. Not sure what is the cost of creating glob but sounds reasonable

Reply:
So let's decide :)

Reply:
What makes more sense for you. I don't mind neither.

Reply:
The question is the cost of glob creation

Reply:
Huuuumph, let's do it in configurations then. So there is no regressions (now we call it only once, let's keep that)